packaging/docker-rh8/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM centos:8
```

Comment:
instead of copying the whole directory, I suggest only copying the dockerfile (and adjusting [the other PR](https://github.com/cloudify-cosmo/pipeline-shared-library/pull/57/files#diff-a56cc437cf229e3d939eb7ac9a1f0d5f646a1464fb44bd1455f617456b37618aR15) to use it), and clearly noting in the comments the lines that changed